mender-dist-packages-building/install-armv6-cross-toolchain

```diff
@@ -0,0 +1,170 @@
+#! /bin/bash
+
+set -ex
+
+# To provide support for Raspberry Pi Zero W a toolchain tuned for ARMv6 architecture must be used.
```

Comment:
Once we are on it, and based on what we have learned, we could improve this comment to be more explicit about the requirement. Something like
```suggestion
# Raspberry Pi OS is compiled for ARMv6 support, so that all RPi boards (including Raspberry Pi Zero W) can run with the same binary image. We are therefore forced to use a toolchain tuned for ARMv6 for our packages
# See the reasoning in this post announcing the 64bit OS: https://www.raspberrypi.com/news/raspberry-pi-os-64-bit/
```

What do you think?